components/org.wso2.carbon.identity.provisioning.connector.google/src/main/java/org/wso2/carbon/identity/provisioning/connector/google/GoogleProvisioningConnectorConfig.java

```diff
@@ -64,12 +64,6 @@ String getUserIdClaim() throws IdentityProvisioningException {
             userIDClaim = "http://wso2.org/claims/streetaddress";
         }
 
-
-        if (StringUtils.isBlank(userIDClaim)) {
-            log.error("UserId cannot mapped to a claim");
```

Comment:
Wouldn't this be an API change?

Reply:
Nope. This condition will not be satisfied in any case